trex.org

```diff
@@ -252,9 +252,14 @@
 * Basis functions
 ** Basis set (basis group)
 
-*** Gaussian-type orbitals
+  Different computational chemistry codes apply normalization at
+  various levels. The normalization of basis functions requires the
+  evaluation of diagonal overlap integrals, which might not always be
+  supported by the client software. For this reason, normalization
+  constants should be explicitly provided in the TREXIO files to
+  ensure portability and completeness.
 
-**** Basis Functions
+*** Gaussian- and Slater-type orbitals
```

Comment:
why the change? this also applies to NAOs, no?

Reply:
Done